embedding/cpp/include/flutter_engine.h

```diff
@@ -0,0 +1,108 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_ENGINE_H_
+#define FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_ENGINE_H_
+
+#include <app_control.h>
+#include <flutter/plugin_registry.h>
+#include <flutter_tizen.h>
+
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+// The engine for flutter execution.
+class FlutterEngine : public flutter::PluginRegistry {
+ public:
+  virtual ~FlutterEngine();
+
+  static std::unique_ptr<FlutterEngine> Create(
+      const std::string& dart_entrypoint = "",
+      const std::vector<std::string>& dart_entrypoint_args = {});
+  static std::unique_ptr<FlutterEngine> Create(
```

Comment:
nit: Adding a blank line between the two `Create` methods might visually help readability.